docs/wiki/Client-Installation-and-Configuration.md

```diff
@@ -77,7 +77,7 @@ DEX client user interface as described in the next section.
 Depending on which assets you wish to use, you have different choices for wallet
 software. There are native/built-in light wallet options for Bitcoin and Decred,
 an external light wallet option for Litecoin, and full-node support for all
-other assets including: Bitcoin, Decred, Litecoin, ZCash, Dogecoin, Bitcoin
+other assets including: Bitcoin, Decred, Litecoin, Zcash, Dogecoin, Bitcoin
```

Comment:
Please put the wiki edit in a separate commit.  I have to keep a sub-tree up-to-date and it's easiest with wiki commits isolated.

Reply:
okay.